doc/releases/release-notes-3.3.rst

```diff
@@ -2592,3 +2592,999 @@ Known Issues
+* :github:`45094` - stm32: Add USB HS device support to STM32H747
+* :github:`44908` - Support ESP32 ADC
+* :github:`44861` - WiFi support for STM32 boards
+* :github:`44410` - drivers: modem: shell: ``modem send`` doesn't honor line ending in modem cmd handler
+* :github:`44399` - Zephyr RTOS support for Litex SoC with 64 bit rocket cpu.
+* :github:`44377` - ISO Broadcast/Receive sample not working with coded PHY
+* :github:`44324` - Compile error in byteorder.h
+* :github:`44318` - boards: arm: rpi_pico: Enable CONFIG_ARM_MPU=y for raspberry pi pico board
+* :github:`44281` - Bluetooth: Use hardware encryption for encryption
+* :github:`44164` - Implement the equivalent of PR #44102 in LLCP
+* :github:`44055` - Immediate alert client
+* :github:`43998` - posix: add include/posix to search path based on Kconfig
+* :github:`43986` - interupt feature for gpio_mcp23xxx
+* :github:`43836` - stm32: g0b1: RTT doesn't work properly after stop mode
+* :github:`43737` - Support compiling ```native_posix`` targets on Windows using the MinGW
+* :github:`43696` - mgmt/mcumgr: RFC: Standardize Kconfig option names for MCUMGR
+* :github:`43655` - esp32c3: Connection fail loop
+* :github:`43647` - Bluetooth: LE multirole: connection as central is not totally unreferenced on disconnection
+* :github:`43604` - Checkpatch: Support in-code ignore tags
+* :github:`43411` - STM32 SPI DMA issue
+* :github:`43330` - usb_dc_nrfx.c starts usbd_work_queue with no name
+* :github:`43308` - driver: serial: stm32: uart will lost data when use dma mode[async mode]
+* :github:`43294` - LoRaWAN stack & user ChannelsMask
+* :github:`43286` - Zephyr 3.1 Release Checklist
+* :github:`42998` - Should board.dts enable peripherals by default?
+* :github:`42910` - Bluetooth: Controller: CIS Data path setup: HCI ISO Data
+* :github:`42908` - Bluetooth: Controller: CIG: LE Remove CIG
+* :github:`42907` - Bluetooth: Controller: CIG: Disconnect: ACL disconnection leading to CIS Disconnection complete
+* :github:`42906` - Bluetooth: Controller: CIG:  Disconnect:  Using HCI Disconnect:  Generate LL_CIS_TERMINATE_IND
+* :github:`42905` - Bluetooth: Controller: CIG: LL Rejects: Remote request being rejected
+* :github:`42902` - Bluetooth: Controller: CIG: Host reject: LE Reject CIS Request
+* :github:`42900` - Bluetooth: Controller: CIG: LE Setup ISO Data Path
+* :github:`42899` - Bluetooth: Controller: CIG: LE CIS Established Event
+* :github:`42898` - Bluetooth: Controller: CIG: LE Accept CIS
+* :github:`42897` - Bluetooth: Controller: CIG: LE CIS Request Event
+* :github:`42896` - Bluetooth: Controller: CIG: LE Create CIS: NULL PDU scheduling
+* :github:`42895` - Bluetooth: Controller: CIG: LE Create CIS: Control procedure with LL_CIS_REQ/RSP/IND PDU
+* :github:`42894` - Bluetooth: Controller: CIG: LE Set CIG Parameters
+* :github:`42700` - Support module.yml in zephyr repo
+* :github:`42653` - [ 2.7.1] STM32 Hangs sending TCP packets
```

Comment:
I don't believe this one belongs in 3.x judging from the comments and commits to fix.  It is focused on the 2.7 LTS

Reply:
Removed